modules/tools/rode/main.tf

```diff
@@ -111,6 +111,33 @@ resource "helm_release" "rode_tfsec_collector" {
   ]
 }
 
+resource "helm_release" "rode_sonarqube_collector" {
+  name       = "rode-collector-sonarqube"
+  namespace  = var.namespace
+  repository = "https://rode.github.io/charts"
+  chart      = "rode-collector-sonarqube"
+  version    = "0.1.0"
+  wait       = true
+
+  set_sensitive {
+    name  = "rode.auth.oidc.clientSecret"
+    value = var.collector_client_secret
+  }
+
+  values = [
+    templatefile("${path.module}/sonar-collector-values.yaml.tpl", {
+      oidc_auth_enabled = var.oidc_token_url != ""
```

Comment:
At one point we talked about making all of these 'enabled' flags explicit instead of implied from the existence of the token url; it looks like we made it explicit for sonarqube but implicit for rode. I could go either way, but wanted to bring it up so we could standardize one way or the other.

Reply:
For now, I just updated it to match the others that use that local.